TodoList/bot/api/constant.ts

```diff
@@ -0,0 +1,15 @@
+export interface User {
+    aadObjectId: string;
+    userName: string;
+    profileImage?: string;
+}
+
```

Comment:
api may not be a good name for all the files. may rename the folder or adjust the folder structure for this project